en/engine/event/event-input.md

```diff
@@ -0,0 +1,100 @@
+# Global System Events
+
+In this section, we will introduce the global system events of Cocos Creator.
```

Comment:
In this section, we will introduce the global system events of Cocos Creator.
==>
In this section, the __global system events__ of **Cocos Creator** will be introduced.